### DISCUSSION TITLE
show_timestamp_in is not parsed as an argument in sb2nov template

### DISCUSSION THREAD

**Discussion by miguelandres** [rendercv/rendercv#282] [Q&A] [answered]
It is used in the template, it's just never parsed.

It also errors out if you have it in your configuration

**Comment by sinaatalay** [comment]
`show_timespan_in` is only available for the `classic` theme. `classic` is the only theme that supports multi-line right columns.

**Reply by miguelandres** [reply]
Understood. I had a bad copy of classic that was named sb2nov locally because of a mistake somewhere. Whoops. My bad. Thanks !

**Reply by sinaatalay** [reply, marked as answer]
`show_timespan_in` is supported for all the themes with v2 now!
